lib/validation/validator.js

```diff
@@ -46,15 +66,18 @@ class Validator {
 			});
 		}
 	}
+
+	static async loadSchema(schemaPath) {
+		const filePath = schemaPath.replace("http://ui5.sap/schema/", "");
+		const schemaFile = await readFile(
+			fileURLToPath(new URL(`./schema/${filePath}`, import.meta.url)), {encoding: "utf8"}
+		);
+		return JSON.parse(schemaFile);
+	}
 }
 
-let validator;
+const validator = {};
```

Comment:
Either use a prototype-less object using `Object.create(null)` or a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)

Matthias changed most of our lists to simple objects in https://github.com/SAP/ui5-project/commit/d4044ecd06dd6fc438a660d5d0af91a46b0e8c62 and related commits. Meanwhile, we try to use Set and Map wherever possible.